scenarioo-server/src/main/java/org/scenarioo/ScenariooViewerApplication.java

```diff
@@ -72,4 +74,16 @@ public void configurePathMatch(PathMatchConfigurer configurer) {
 	public void configureContentNegotiation(ContentNegotiationConfigurer configurer) {
 		configurer.favorPathExtension(false);
 	}
+
+	/**
+	 * Spring Boot uses the StrictHttpFirewall. By default it blocks requests containing URL encoded percent signs.
+	 * In the Scenarioo End to End Tests this occurs in the scenario "Use breadcrumbs > Tooltip in breadcrumbs".
+	 * If this feature is not disabled then an Internal Server Error is shown to the user.
+	 */
+	@Bean
+	public HttpFirewall allowUrlEncodedPercentHttpFirewall() {
```

Comment:
Thanks for checking this! I was wondering, wether some other properties of the default firewall would get overwritten by instantiating a new one here. Do yoiu know, if we could also put this in our default configuration file?

Reply:
I didn't find an option to configure this in the configuration file.